### ISSUE TITLE
feat: add telescope extension to view all icons and its color

### ISSUE THREAD

**Issue by WieeRd** [nvim-tree/nvim-web-devicons#252]
A telescope picker that can list all the icons with associated filetype/filename and color code.  
Not something very necessary, but it would be cool to browse through all the icons available.

**Comment by gegoune** [comment]
I don't think we should be adding support to all available pickers out there. 

There is `require('nvim-web-devicons').get_icons()` API that can be used to implement it outside of this repo.

Will close it for now, but will be happy to reopen if others think such functionality belongs here.
